environment.yml

```diff
@@ -0,0 +1,19 @@
+name: NAB
+channels:
+  - defaults
+  - conda-forge
+
+dependencies:
+  - python=3.6
+  - pip
+
+  # See requirements.txt
+  - pandas==0.20.3
+  - simplejson==3.11.1
+  - boto3==1.9.134
+  - scikit-learn==0.21.1
+
+  - pip:
+    - boto3
+    - botocore
+
```

Comment:
You should add NAB itself to `- pip:` section.
Something like this:
```
- pip:
    - boto3
    - botocore
    # Install NAB in development mode
    - -e .
```
See https://github.com/numenta/nupic.research/blob/master/environment.yml#L69-L70